extensions/vertx-web/deployment/src/main/java/io/quarkus/vertx/web/deployment/VertxWebProcessor.java

```diff
@@ -194,6 +196,20 @@ public void transform(TransformationContext context) {
         });
     }
 
+    @BuildStep
+    WebsocketSubProtocolsBuildItem websocketSubProtocols(CombinedIndexBuildItem combinedIndexBuildItem) {
+        return isApolloInClasspath() ? new WebsocketSubProtocolsBuildItem("graphql-ws") : null;
```

Comment:
This should be done in the Graph QL extension itself.

Reply:
vertx-web-graphql is part of vert-web, do we need another quarkus extension for that? just for graphql.

Reply:
So there is not going to be a seperate graphql extension? In that case I think this is ok.

Reply:
I dont think it is needed.

Reply:
I had a look at the vert.x module and I think this definitely needs its own extension, if only to provide config. I will also be very surprised if this compiles to native without an extension to register stuff for reflection..